types/geakstr__sass-inline-svg/geakstr__sass-inline-svg-tests.ts

```diff
@@ -0,0 +1,10 @@
+import inliner from '@geakstr/sass-inline-svg';
```

Comment:
Use the commonjs syntax `import inliner = require('inliner')`.

Consumers of the package who want to use `import inliner from 'inliner'` will need to add `"esModuleInterop": true` or use ES modules, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.